tests/DependencyInjection/CompilerPass/OverrideServiceCompilerPassTest.php

```diff
@@ -39,18 +39,18 @@ protected function setUp(): void
      */
     public function testShouldRegisterTwigPaths(array $paths, int $expectedCalls): void
     {
-        $this->builder->setParameter('spirit_web_twig.paths', $paths);
-        $this->builder->setParameter('spirit_web_twig.paths_alias', 'test');
-        $this->builder->setParameter('spirit_web_twig.html_syntax_lexer', false);
-        $this->builder->setParameter('spirit_web_twig.spirit_css_class_prefix', null);
-        $this->builder->setParameter('spirit_web_twig.icons.paths', []);
-        $this->builder->setParameter('spirit_web_twig.icons.alias', 'test-icons');
+        $this->builder->setParameter('twigx.paths', $paths);
+        $this->builder->setParameter('twigx.paths_alias', 'test');
+        $this->builder->setParameter('twigx.html_syntax_lexer', false);
+        $this->builder->setParameter('twigx.css_class_prefix', null);
+        $this->builder->setParameter('twigx.icons.paths', []);
```

Comment:
it is desirable to be able to turn off the html-like syntax by twigx.html_syntax_lexer parameter?

Reply:
Ok, I've removed that configuration.